sympy/stats/frv.py

```diff
@@ -375,12 +372,120 @@ class SingleFinitePSpace(SinglePSpace, FinitePSpace):
     def domain(self):
         return SingleFiniteDomain(self.symbol, self.distribution.set)
 
+    @property
+    def _is_symbolic(self):
+        """
+        Helper property to check if the distribution
+        of the random variable is symbolic.
+        """
+        return isinstance(self.domain.set, Intersection)
+
+    def _is_logical(self, expr):
+        """
+        Helper function to check if an expression
+        is logical.
+
+        Returns
+        =======
+
+        True
+            If the expression is logical.
+        False
+            In all other cases.
+        """
+        return isinstance(expr, (Relational, Logic))
```

Comment:
This method doesn't seem necessary

Reply:
Yes, I just added it for clearer meaning. Well, I will remove this.